src/Message/Reset.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the user bundle package.
+ * (c) Connect Holland.
+ */
+
+namespace ConnectHolland\UserBundle\Message;
+
+use ApiPlatform\Core\Annotation\ApiResource;
```

Comment:
Can the scrutinizer issue be fixed?
The type ApiPlatform\Core\Annotation\ApiResource was not found.

Reply:
No, because the API platform is a suggested package, but not a required package. The alternative was creating a separate package for api platform in combination with the user bundle.